Library/Homebrew/cmd/update-report.rb

```diff
@@ -113,7 +113,7 @@ def update_report
     end
 
     if !updated
-      puts "Already up-to-date." if !args.preinstall? && !ENV["HOMEBREW_UPDATE_FAILED"]
+      puts "Homebrew already up-to-date." if !args.preinstall? && !ENV["HOMEBREW_UPDATE_FAILED"]
```

Comment:
Can we make this text change in another PR? Thanks!

Reply:
Why I changed it is because it looks really weird when you first get to see text about completions being updated and then text that says "Already up-to-date." That feels rather confusing.

Reply:
I don't think the completion being updated is important enough to show a message. It should just happen.

Reply:
That could be an option, too.

Or I could have look at how to get it into the update report at the end.

Reply:
I removed the text output for completion updating and reverted the "up-to-date" text.